pkg/client/cli/cliutil/systema.go

```diff
@@ -59,23 +59,24 @@ func EnsureLoggedOut(ctx context.Context) error {
 // HasLoggedIn returns true if either the user has an active login session or an expired login
 // session, and returns false if either the user has never logged in or has explicitly logged out.
 func HasLoggedIn(ctx context.Context) bool {
-	token, _ := authdata.LoadTokenFromUserCache(ctx)
-	return token != nil
+	userInfo, _ := authdata.LoadUserInfoFromUserCache(ctx)
```

Comment:
Do we want this error to go undetected?

Reply:
We do.  On the invariant that `(userInfo == nil) != (err == nil)`, we only need to look at one.  Perhaps I should have used `_, err := …; return err == nil`.